scripts/shared/resources/nexd-kustomization.yaml

```diff
@@ -0,0 +1,37 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+namespace: nexodus
+bases:
+  - ../../nexodus/deploy/nexodus-client/base
+secretGenerator:
+  - name: nexodus-client-secret
+    literals:
+      - cert=${NEXODUS_CERT}
+      - url=https://try.nexodus.127.0.0.1.nip.io
+      - auth_url=https://auth.try.nexodus.127.0.0.1.nip.io/token
+      - username=kitteh1
+      - password=floofykittens
+commonLabels:
+  app.kubernetes.io/component: nexodus-client
+  app.kubernetes.io/instance: nexodus-client
+  app.kubernetes.io/name: nexodus-client
+patches:
+  - target:
+      kind: DaemonSet
+      name: nexodus
+    patch: |-
+      - op: replace
+        path: /spec/template/spec/hostAliases/0/ip
+        value: ${HOST_IP}
+      - op: replace
+        path: /spec/template/spec/containers/0/command
+        value:
+          - /bin/sh
+          - -c
+          - |
+            CAROOT=/etc/nexodus/.certs /bin/mkcert -install
+            /bin/nexd --username=\$USERNAME --password=\$PASSWORD --service-url=\$URL \\
+            router --disable-nat "${NEXODUS_CIDRS[@]}"
```

Comment:
Do we really need the `--disable-nat`. Its use-case seems to be different.

Reply:
> By default, Nexodus network routers perform NAT, specifically, source NAT for devices coming from a Nexodus mesh with a destination of one of the devices not running the Nexodus agent. This enables connectivity to those devices without any configuration on the devices.
> 
> You have the option to disable NAT with --disable-nat which will cause the remote non-Nexodus devices to receive traffic from the Nexodus agent devices without any address translations. This mode requires routes to be added (or redistributed in your network IGP) for hosts in 192.168.1.0/24 to reach Nexodus nodes 100.100.0.0/16 via the Nexodus Network Router eth0 ip of 192.168.1.10.